ribs.py

```diff
@@ -123,6 +123,16 @@ def clear_screen(color):
     bottom_right = pg.display.get_surface().get_size()
     pg.draw.rect(window, color, (top_left, bottom_right))
 
+
+def draw_text(text, position, size, color=pg.Color(255, 255, 255), font_name=None):
+
+    font = pg.font.SysFont(font_name, size)
+
+    rendered_text = font.render(text, True, color)
+
+    window = pg.display.get_surface()
```

Comment:
Maybe we should extract this to a different function? Or is this call easy enough to understand?

Reply:
I don't think it is too complicated. I was thinking about adding a set_default_font and maybe a set_text_color to set the defaults instead of passing them along each time. Maybe even a default size?

Reply:
The rest of the code base doesn't use hidden state, we took an active decision to not require syncing and knowing the state of the framework. I'd have default parameters, just to make it easier to use.